### ISSUE TITLE
installation of v2.143.0 rpm from advisory=FEDORA-2020-04374575bf doesn't seem to be using updated policy from this repo

### ISSUE THREAD

**Issue by dweomer** [containers/container-selinux#101]
Trying to validate the RPM installation on Fedora 32 for the recently tagged v2.143.0 as described [on bodhi](https://bodhi.fedoraproject.org/updates/FEDORA-2020-04374575bf):
```
dnf upgrade --enablerepo=updates-testing --advisory=FEDORA-2020-04374575bf
```
**_And it does not seemed to have picked up the policy in this repo._** 

This Vagrantfile is my validation harness: https://github.com/containerd/containerd/blob/586061508e1dd1ba5305778265f967ba5a409bcc/Vagrantfile
After spinning up the VM via `vagrant up` I connect to it and run the installation line from bodhi, and seemingly have the correct RPM via `dnf info container-selinux`:
```
Last metadata expiration check: 0:02:59 ago on Fri 07 Aug 2020 11:39:21 PM UTC.
Installed Packages
Name         : container-selinux
Epoch        : 2
Version      : 2.143.0
Release      : 1.fc32
Architecture : noarch
Size         : 45 k
Source       : container-selinux-2.143.0-1.fc32.src.rpm
Repository   : @System
From repo    : updates-testing
Summary      : SELinux policies for container runtimes
URL          : https://github.com/containers/container-selinux
License      : GPLv2
Description  : SELinux policy modules for use with container runtimes.
```
I then run the `test-cri` provisioner with SELinux Enforcing invoking:
```
SELINUX=Enforcing vagrant up --provision-with=selinux,test-cri
```
This starts containerd and then runs critest against it. Prior to my fix that should be in v2.143.0 I would see critest suite failures like this:
```
    default: Summarizing 11 Failures:
    default: 
    default: [Fail] 
    default: [k8s.io] Security Context NamespaceOption [It] runtime should support HostPID 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:389
    default: [Fail] [k8s.io] Security Context NamespaceOption [It] runtime should support ContainerPID 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:389
    default: 
    default: [Fail] [k8s.io] Multiple Containers [Conformance] when running multiple containers in a pod [It] should support container exec 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:389
    default: 
    default: [Fail] [k8s.io] Security Context NamespaceOption [It] runtime should support PodPID 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/container.go:389
    default: 
    default: [Fail] 
    default: [k8s.io] Streaming runtime should support streaming interfaces [It] runtime should support portforward in host network 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/networking.go:253
    default: 
    default: [Fail] [k8s.io] Multiple Containers [Conformance] when running multiple containers in a pod 
    default: [It] should support network 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/networking.go:253
    default: 
    default: [Fail] [k8s.io] Networking runtime should support networking [It] runtime should support port mapping with host port and container port [Conformance] 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/networking.go:253
    default: 
    default: [Fail] [k8s.io] Security Context NamespaceOption 
    default: [It] runtime should support HostIpc is true 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/security_context_linux.go:159
    default: 
    default: [Fail] [k8s.io] Streaming runtime should support streaming interfaces [It] runtime should support portforward [Conformance] 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/networking.go:253
    default: 
    default: [Fail] 
    default: [k8s.io] Networking runtime should support networking [It] runtime should support port mapping with only container port [Conformance] 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/networking.go:253
    default: 
    default: [Fail] [k8s.io] Multiple Containers [Conformance] when running multiple containers in a pod [It] should support container log 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/multi_container_linux.go:95
    default: 
    default: Ran 88 of 95 Specs in 246.619 seconds
    default: FAIL! -- 77 Passed | 11 Failed | 0 Pending | 7 Skipped
```
**And this is what I still see with this RPM installed. This is surprising to me**. When I install the policy from this repo via source (after installing `selinux-policy-devel` and `bzip2`) by running `make && sudo make install-policy` I get the results that I expect, e.g.:
```
    default: Summarizing 1 Failure:
    default: 
    default: [Fail] [k8s.io] Security Context NamespaceOption 
    default: [It] runtime should support HostIpc is true 
    default: /root/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/security_context_linux.go:159
    default: 
    default: Ran 88 of 95 Specs in 65.534 seconds
    default: FAIL! -- 87 Passed | 1 Failed | 0 Pending | 7 Skipped
```
Is there something wrong with how the RPM is being build? This build log seems to indicate that the wrong tag is being used: https://kojipkgs.fedoraproject.org//packages/container-selinux/2.143.0/1.fc32/data/logs/noarch/build.log

**Comment by dweomer** [comment]
Yeah, this prep section of the [build log](https://kojipkgs.fedoraproject.org//packages/container-selinux/2.143.0/1.fc32/data/logs/noarch/build.log) indicates that it is building the v2.139.0 tag:
```
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.EEmaQ5
+ umask 022
+ cd /builddir/build/BUILD
+ cd /builddir/build/BUILD
+ rm -rf container-selinux-8c26927bc43112dce4088a453a1c446c3a15c21f
+ /usr/bin/gzip -dc /builddir/build/SOURCES/container-selinux-8c26927.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd container-selinux-8c26927bc43112dce4088a453a1c446c3a15c21f
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ /usr/bin/git init -q
+ /usr/bin/git config user.name rpm-build
+ /usr/bin/git config user.email '<rpm-build>'
+ /usr/bin/git add .
+ /usr/bin/git commit -q --allow-empty -a --author 'rpm-build <rpm-build>' -m 'container-selinux-2.143.0 base'
+ RPM_EC=0
++ jobs -p
+ exit 0
```

**Comment by dweomer** [comment]
This commit to the f32 branch only updated the version, not the actual commit: https://src.fedoraproject.org/rpms/container-selinux/c/0e7556f827f085e511a5709469cf5352bf39fc47?branch=f32
```
[root@3eb042ad4575 container-selinux]# git status
On branch f32
Your branch is up to date with 'origin/f32'.

nothing to commit, working tree clean
[root@3eb042ad4575 container-selinux]# git log -1
commit 0e7556f827f085e511a5709469cf5352bf39fc47 (HEAD -> f32, origin/f32)
Author: RH Container Bot <rhcontainerbot@fedoraproject.org>
Date:   Wed Aug 5 22:10:44 2020 +0000

    container-selinux-2:2.143.0-1
    
    Signed-off-by: RH Container Bot <rhcontainerbot@fedoraproject.org>
```
`[root@3eb042ad4575 container-selinux]# git diff master -- container-selinux.spec`:
```patch
diff --git a/container-selinux.spec b/container-selinux.spec
index be5eb52..7d3621c 100644
--- a/container-selinux.spec
+++ b/container-selinux.spec
@@ -2,7 +2,7 @@
 
 # container-selinux
 %global git0 https://github.com/containers/container-selinux
-%global commit0 e2d5a9eadb72a9aa90c4f5ba793011865620f367
+%global commit0 8c26927bc43112dce4088a453a1c446c3a15c21f
 %global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
 
 # container-selinux stuff (prefix with ds_ for version/release etc.)
@@ -17,15 +17,17 @@
 %global _format() export %1=""; for x in %{modulenames}; do %1+=%2; %1+=" "; done;
 
 # Version of SELinux we were using
-%global selinux_policyver 3.14.4-43
+%if 0%{?fedora}
+%define selinux_policyver 3.14.4-43
+%else
+%define selinux_policyver 3.14.3-20
+%endif
 
 # Hooked up to autobuilder, please check with @lsm5 before updating
 Name: container-selinux
-%if 0%{?fedora}
 Epoch: 2
-%endif
 Version: 2.143.0
-Release: 2.dev.git%{shortcommit0}%{?dist}
+Release: 1%{?dist}
 License: GPLv2
 URL: %{git0}
 Summary: SELinux policies for container runtimes
@@ -44,7 +46,7 @@ Requires(post): libselinux-utils
 Requires(post): sed
 Obsoletes: %{name} <= 2:1.12.5-13
 Obsoletes: docker-selinux <= 2:1.12.4-28
-Provides: docker-selinux = %{?epoch:%{epoch}:}%{version}-%{release}
+Provides: docker-selinux = %{epoch}:%{version}-%{release}
 
 %description
 SELinux policy modules for use with container runtimes.
@@ -64,9 +66,6 @@ install -p -m 644 container.if %{buildroot}%{_datadir}/selinux/devel/include/ser
 install -m 0644 $MODULES %{buildroot}%{_datadir}/selinux/packages
 install -d %{buildroot}/%{_datadir}/containers/selinux
 install -m 644 container_contexts %{buildroot}/%{_datadir}/containers/selinux/contexts
-# Currently shipped as part of selinux-policy package
-#install -d %{buildroot}/%{_datadir}/man/man8
-#install -m 644 container_selinux.8 %{buildroot}/%{_datadir}/man/man8
 
 # remove spec file
 rm -rf container-selinux.spec
@@ -104,108 +103,88 @@ fi
 %files
...skipping...
-- autobuilt f00d1f4
+* Mon Apr 06 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.130.0-1
+- autobuilt v2.130.0
 
-* Sun Mar 29 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.128.0-2.dev.git363646f
-- bump to 2.128.0
-- autobuilt 363646f
+* Mon Mar 30 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.129.0-1
+- autobuilt v2.129.0
 
-* Fri Mar 27 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.127.0-2.dev.git6caf15d
-- bump to 2.127.0
-- autobuilt 6caf15d
+* Mon Mar 30 2020 Dan Walsh <dwalsh@fedoraproject.org> - 2:2.126.0-2
+- Fix commmit version
 
-* Thu Mar 26 2020 Dan Walsh <dwalsh@fedoraproject.org> - 2:2.126.0-2.dev.git867a377
-- Install selinux contexts file into /usr/share/containers/selinux/contexts
+* Sun Mar 29 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.129.0-1
+- autobuilt v2.129.0
 
-* Thu Mar 26 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.126.0-2.dev.git867a377
-- bump to 2.126.0
-- autobuilt 867a377
+* Sun Mar 29 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.128.0-1
+- autobuilt v2.128.0
 
-* Mon Mar 23 2020 Lokesh Mandvekar <lsm5@fedoraproject.org> - 2:2.125.2-2.dev.gitae0720d
-- bump release tag
+* Fri Mar 27 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.127.0-1
+- autobuilt v2.127.0
 
-* Mon Mar 23 2020 Dan Walsh <dwalsh@fedoraproject.org> - 2:2.125.2-1.dev.gitae0720d
+* Thu Mar 26 2020 Dan Walsh <dwalsh@fedoraproject.org> - 2:2.126.0-1
+- Add container_kvm_t for kata containers
+- Add contaienr_init_t for systemd based containers
 - Install container_contexts file
 
-* Mon Mar 23 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.125.0-3.1.dev.gitfde876b
-- autobuilt fde876b
+* Mon Mar 23 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.125.2-1
+- autobuilt v2.125.2
 
-* Mon Mar 23 2020 Lokesh Mandvekar <lsm5@fedoraproject.org> - 2:2.125.0-2.1.dev.gitb321ea4
-- bump release tag for smooth upgrade path
+* Mon Mar 23 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.125.1-1
+- autobuilt v2.125.1
 
-* Fri Mar 20 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.125.0-0.1.dev.gitb321ea4
-- bump to 2.125.0
-- autobuilt b321ea4
+* Fri Mar 20 2020 RH Container Bot <rhcontainerbot@fedoraproject.org> - 2:2.125.0-1
+- autobuilt v2.125.0
 
 * Tue Feb 11 2020 Lokesh Mandvekar <lsm5@fedoraproject.org> - 2:2.124.0-4.dev.git5624558
 - keep functional upgrade path from f31
```

**Comment by dweomer** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1775831#c24

**Comment by dweomer** [comment]
fixed with 2.144.0
